### PR TITLE
Blog: Enhancing Acknowledgment of Community Contributions in OpenTelemetry Profiling

### PR DESCRIPTION
**Pull request by Rperry2174**
This PR introduces modifications to the blog post about OpenTelemetry's profiling support to better acknowledge the contributions of various community members and organizations.  The key changes include:

Expanded Acknowledgements: We're enhancing the acknowledgements section to highlight the collaborative efforts within the Profiling SIG.

Inclusive Recognition: Specifically mentioning contributions from companies like Grafana Labs, Datadog, and Red Hat to recognize their significant input

Highlighting Community Contributions:

- Mentioning the proposals for the profiling data model (OTEPs 237 and 239)
- Acknowledging the broad-based discussion and alignment within the community on the profiling data format.

These changes aim to reflect the collective and diverse efforts that have been instrumental in advancing OpenTelemetry's profiling capabilities, ensuring that more key contributors are recognized and it's clearer that this is a community effort

More discussion on this in the thread [here](https://cloud-native.slack.com/archives/C03J794L0BV/p1710851315755469)
relevant prs: https://github.com/open-telemetry/opentelemetry.io/pull/4183/

### CODE REVIEW

**Review by austinlparker — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
Thanks! This is a good addition.

**Review by austinlparker — CHANGES_REQUESTED** [review]
temp hold on this

**Comment by cartermp** [comment]
yee yee

**Comment by disfluxly** [comment]
This is definitely needed. As a community member that has followed this pretty closely, I was very surprised not seeing Ryan and/or Grafana Labs anywhere in the original post.

**Comment by tedsuo** [comment]
Sorry to fuss over a blog post, Is it possible to list the people, not the companies? In OTel we prefer to highlight individual contributions, it sometimes ruffle feathers when public communication from the project can be interpreted as an advertisement for certain companies. 

If it's a big donation it's one thing, but when we're talking about SIG contributions we try to focus on individuals.

File: content/en/blog/2024/opentelemetry-announced-support-for-profiling.md [hunk 1]
```diff
@@ -12,8 +12,34 @@ vision of enabling built-in observability for cloud native applications requires
 us to continue evolving with the community. This year, we’re proud to announce
 that exactly two years after the Profiling SIG was created at KubeCon +
 CloudNativeCon Europe 2022 in Valencia, we’re taking a big step towards this
-goal by merging a profiling data model into our specification and working
-towards a stable implementation this year!
+goal by merging a profiling data model OTEP and working towards a stable spec
+and implementation this year!
+
+This milestone for OpenTelemetry profiling feature reflects a collaborative
```
Comment (felixge) [inline content/en/blog/2024/opentelemetry-announced-support-for-profiling.md:18]: grammar nit

```suggestion
This milestone for the OpenTelemetry profiling signal reflects a collaborative
```

**Review by felixge — APPROVED** [review]
Thanks Ryan 🙇

**Comment by felixge** [comment]
> Sorry to fuss over a blog post, but is it possible to list the people, not the companies? In OTel we prefer to highlight individual contributions, it sometimes ruffle feathers when public communication from the project can be interpreted as an advertisement for certain companies.
> 
> If it's a big donation it's one thing, but when we're talking about SIG contributions we try to focus on individuals.

How about naming individuals with their company names in parens? Otherwise this approach raises questions about the value of SIG work vs code donations.

**Review by theletterf — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]
lgtm

**Comment by tedsuo** [comment]
@felixge thats totally fine, it's usually how we do it.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**